run.py

```diff
@@ -368,6 +368,9 @@ def check_requires(requires, suricata_config: SuricataConfig):
         elif key == "lambda":
             if not eval(requires["lambda"]):
                 raise UnsatisfiedRequirementError(requires["lambda"])
+        elif key == "os":
```

Comment:
@jasonish I want to check the OS that a test is being run on and skip if it isn't the expected one. Do you think this is a good way of testing that? Is there a better way?

Reply:
I guess `platform.system()` is known to have a little more consistent output.  So I'd use that.. `os` is good for the configuration key (remember to add to readme).  May as well do a lower case conversion before comparing as well, as `platform.system()` returns "Linux", "Windows", etc.

Reply:
Thank you! Done in the following PR.